addOns/oast/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Fixed
+- Deregister the Interactsh service even in case of error (Issue 7504).
```

Comment:
Although this message is okay, this is not what is actually happening. We are simply marking the service as deregistered  on our side after attempting to deregister once, regardless of whether it is actually deregistered on the server.

Reply:
What do you suggest should be the message then?

Reply:
The message is okay, just wanted to point it out for the record / completeness :).